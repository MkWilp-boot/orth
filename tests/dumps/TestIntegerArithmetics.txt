100
100
-30
0
10
45