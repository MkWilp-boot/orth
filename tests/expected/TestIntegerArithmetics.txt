100
100
30
5
0
5